### PR TITLE
kertish-dfs 21.2.0066 (new formula)

### PR DESCRIPTION
**Pull request by freakmaxi**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/kertish-dfs.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      system "go", "build", *std_go_args, "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtfs"
+    end
+    cd "admin-tool" do
+      system "go", "build", *std_go_args, "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtadm"
+    end
+  end
+
+  test do
+    assert_match(/failed.\nlocalhost:39400: head node is not reachable\n$/,
+      `#{bin}/krtfs -t localhost:39400 ls`)
```
Comment (SMillerDev) [inline Formula/kertish-dfs.rb:22]:
```suggestion
      shell_output("#{bin}/krtfs -t localhost:39400 ls"))
```

File: Formula/kertish-dfs.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    assert_match(/localhost:39400: manager node is not reachable\n$/,
+      `#{bin}/krtadm -t localhost:39400 -get-clusters`)
```
Comment (SMillerDev) [inline Formula/kertish-dfs.rb:24]:
```suggestion
      shell_output("#{bin}/krtadm -t localhost:39400 -get-clusters"))
```

File: Formula/kertish-dfs.rb (hunk 1 above)
Comment (freakmaxi) [reply Formula/kertish-dfs.rb:22]: Tests are failing with this way

File: Formula/kertish-dfs.rb (hunk 2 above)
Comment (freakmaxi) [reply Formula/kertish-dfs.rb:24]: Tests are failing with this way

File: Formula/kertish-dfs.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/kertish-dfs.rb:22]: You can add the expected returncode as the second argument of the function if it is not 0.

File: Formula/kertish-dfs.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    cd "admin-tool" do
+      system "go", "build", *std_go_args, "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtadm"
+    end
+  end
+
+  test do
+    assert_match(/failed.\nlocalhost:39400: head node is not reachable\n$/,
+      shell_output("#{bin}/krtfs -t localhost:39400 ls"))
+    assert_match(/localhost:39400: manager node is not reachable\n$/,
+      shell_output("#{bin}/krtadm -t localhost:39400 -get-clusters", 70))
```
Comment (SMillerDev) [inline Formula/kertish-dfs.rb:24]: The regex was not really required. And this way we're sure the port isn't being used by another tool
```suggestion
   port = free_port
    assert_match("failed.\nlocalhost:#{port}: head node is not reachable",
      shell_output("#{bin}/krtfs -t localhost:#{port} ls"))
    assert_match("localhost:#{port}: manager node is not reachable",
      shell_output("#{bin}/krtadm -t localhost:#{port} -get-clusters", 70))
```

Comment (freakmaxi) [reply Formula/kertish-dfs.rb]: It is a good point!

**Review by SMillerDev — DISMISSED** [review]
Thanks @freakmaxi  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by SMillerDev** [comment]
Please squash all your commits into one, using the title of the PR as commit message. Our workflow doesn't like merge commits.

**Comment by freakmaxi** [comment]
> Please squash all your commits into one, using the title of the PR as commit message. Our workflow doesn't like merge commits.

I do not have anything else to squash. It is already squash... If you want, I can close this PR and open a new one. What do you say?

**Pull request closed without merging**